features/support/parameter_types.rb

```diff
@@ -150,7 +144,11 @@ def provider_by_name(name)
   name: 'buyer',
   type: Account,
   regexp: /buyer "([^"]*)"|the buyer/,
-  transformer: ->(org_name) { org_name.present? ? Account.buyers.find_by!(org_name: org_name) : @buyer }
+  transformer: ->(org_name) do
+    return Account.buyers.find_by!(org_name: org_name) if org_name.present?
+
+    @buyer || @account
+  end
```

Comment:
do we ignore buyer name or do I miss something?

Reply:
I don't know, this comes from way before and it will be refactored.